payment_signal/models/inherit_res_company.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    default_signal = fields.Integer('Percent Signal in S.O.',
+                                    default=20,
+                                    store=True)
```

Comment:
You should not use store=True not fields which are not computed, default they are stored